### ISSUE TITLE
Trader places orders with incorrect amounts for Asset and Currency.

### ISSUE THREAD

**Issue by hilkoc** [askmike/gekko#1723]
**Note: for support questions, please join our [Discord server](https://discord.gg/26wMygt)**

* **I'm submitting a ...**
[x ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)
Run a live trade bot to trade XBTEUR on Kraken

* **Expected result** (what you hoped would happen)
Orders executed correctly.

* **Actual result** (unexpected outcome)
Gekko computes the wrong amount leading to errors

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

I've tried running a real tradebot with some of the build-in strategies, DEMA, MACD and my own test strategy that simply buys and sells at a fixed level.

Here's the relevant log output
```
2018-01-18 20:31:52 (INFO):	trading at kraken ACTIVE
2018-01-18 20:31:52 (INFO):	kraken trading fee will be: 0.16%
2018-01-18 20:31:52 (INFO):	kraken portfolio:
2018-01-18 20:31:52 (INFO):		 EUR: 10.684500000000
2018-01-18 20:31:52 (INFO):		 XBT: 0.047447450000
2018-01-18 20:31:52 (INFO):	Setting up:
2018-01-18 20:31:52 (INFO):		 Performance Analyzer
2018-01-18 20:31:52 (INFO):		 Analyzes performances of trades
2018-01-18 20:31:52 (INFO):	
...
2018-01-18 20:38:12 (DEBUG):	calculated DEMA properties for candle:
2018-01-18 20:38:12 (DEBUG):		 long ema: 9909.02393396
2018-01-18 20:38:12 (DEBUG):		 short ema: 9905.11993657
2018-01-18 20:38:12 (DEBUG):		 diff: -0.03941
2018-01-18 20:38:12 (DEBUG):		 DEMA age: 22 candles
2018-01-18 20:38:12 (DEBUG):	we are currently in a downtrend @ 9900.00000000 (-0.03941)
2018-01-18 20:38:12 (INFO):	Trader Received advice to go short. Selling  XBT
2018-01-18 20:38:13 (DEBUG):	[kraken.js] entering "setBalance" callback after kraken-api call, data: { error: [],
  result: 
   { ZUSD: '0.0000',
     ZEUR: '10.6845',
     XXBT: '0.0474474500',
     XXRP: '0.00147235' } }
2018-01-18 20:38:13 (ERROR):	Wanted to buy EUR but the amount is too small (0.047447450000) at kraken
```
So the trader receives the advice to go short. Selling XBT at 9900. Since the XBT balance at start is 0.04744745, the amount of EUR to buy must be:
9900 x 0.04744745 = 469.73.

Instead, we get the error: "Wanted to buy EUR but the amount is too small (0.047447450000) at kraken"
So it uses the amount of XBT instead of the amount of EUR, leading to an error.

Has anyone seen this problem before? I really appreciate any help.

**Comment by bitkominer** [comment]
I think it's correct amount. The one you need to modify is the list of minimum amount mentioned inside file: kraken.js (find it in folder: \exchanges\).

`
 //Tradeable against XBT
      { pair: ['BCH', 'XBT'], minimalOrder: { amount: 0.01, unit: 'asset' }, precision: 5 },
      { pair: ['CAD', 'XBT'], minimalOrder: { amount: 0.1, unit: 'asset' }, precision: 1 },
      { pair: ['EUR', 'XBT'], minimalOrder: { amount: 0.1, unit: 'asset' }, precision: 1 },
      { pair: ['GBP', 'XBT'], minimalOrder: { amount: 0.01, unit: 'asset' } },
      { pair: ['JPY', 'XBT'], minimalOrder: { amount: 1, unit: 'asset' }, precision: 0 },
      { pair: ['USD', 'XBT'], minimalOrder: { amount: 0.1, unit: 'asset' }, precision: 1 },
    ],
`

See the above list, minimum order for pair EUR-XBT is 0.1, while your XBT balance is only 0.047. Just modify the minimum order into less than your balance, e.g: 
` { pair: ['EUR', 'XBT'], minimalOrder: { amount: 0.01, unit: 'asset' }, precision: 1 },`

**Comment by hilkoc** [comment]
Thanks @bitkominer . I was looking through the src and indeed I see this error text is raised after checking the minimum amount.
I did an update to the latest of the develop branch. This pulled in the correct minimum amounts for Kraken, 0.02 XBT. 
But now I can't see if it actually works because a new problem occurred.  I can't see any logging when running a live tradebot from the UI?  Before I pulled in the changes it would write to a file in the logs/ directory. Now there's nothing there anymore and the logs with trade advice don't show in the console.

Any idea why this can be or how to fix it?
